js/techtree.js

```diff
@@ -368,16 +368,17 @@ class Tree {
         };
         this.height = Math.max(window.innerHeight - 80, 100);
         this.width = 0;
-        this.padding = 10;
+        this.padding = 20;
         this.element_height = 0;
         this.lanes = [];
+        this.offset_x = 150;  // 150 is starting offset from the left to accommodate age icons
     }
 
     updateOffsets() {
         this.element_height = this.height / 4 / 3;
         let element_offset = this.element_height / 2;
 
-        this.offsets.dark_1 = this.padding;
+        this.offsets.dark_1 = this.padding;  // shift the tech tree down by 10px to align center in the row
```

Comment:
This comment is obsolete now :slightly_smiling_face:

Reply:
Nuked it!!